### ISSUE TITLE
`wrap` uses `pbc` instead of `wrap` keyword

### ISSUE THREAD

**Issue by lilyminium** [MDAnalysis/mdanalysis#3909]
## Expected behavior ##

Using `AtomGroup.wrap` as intended does not trigger a DeprecationWarning.

## Actual behavior ##

It does here https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/files/ because of `center_of_mass(pbc=False)` and `center_of_geometry(pbc=False)` calls, which use the old `pbc` keyword instead of `wrap`.

## Code to reproduce the behavior ##

``` python
import MDAnalysis as mda
from MDAnalysis.tests.datafiles import PSF, DCD,  GRO, PDB, TPR, XTC, TRR,  PRMncdf, NCDF

u = mda.Universe(TPR, XTC)

not_protein = u.select_atoms('not protein')

for ts in u.trajectory:
    not_protein.wrap(compound='residues')

```
```
 /home/pbarletta/anaconda3/envs/mda-user-guide/lib/python3.8/site-packages/MDAnalysis/core/groups.py:440: DeprecationWarning: The 'pbc' kwarg has been deprecated and will be removed in version 3.0., please use 'wrap' instead 
  warnings.warn("The 'pbc' kwarg has been deprecated and will be " 
```

## Current version of MDAnalysis ##

- Which version are you using? (run `python -c "import MDAnalysis as mda; print(mda.__version__)"`) develop
- Which version of Python (`python -V`)? 3.8
- Which operating system?
